.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,81 @@
+name: Unit Tests
+
+on:
+  # Run on all pushes and on all pull requests.
+  # Prevent the "push" build from running when there are only irrelevant changes.
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: PHP ${{ matrix.php }}
+    # ubuntu-20.04 / ubuntu-latest includes MySQL 8, which has issues with older versions of PHP.
+    # ubuntu-18.04 includes PHP versions 7.1-8.0, but 5.6-7.1 are cached, so setup is about 5 seconds.
+    # See https://setup-php.com/i/452
+    runs-on: ubuntu-18.04
```

Comment:
What about running it on ubuntu-20.04? Do we still have issues with older versions of PHP? Same question for the integration tests.

Reply:
The issue was with MySQL 8 - I can't remember the specifics, and right now the hassle isn't worth it, as 18.04 provides everything we need.

Once we move to a minimum of PHP 7.1, _then_ we can look again at bumping to ubuntu-20.04.

Reply:
That makes sense. I've opened an issue for when we release 3.0. https://github.com/Parsely/wp-parsely/issues/413